src/shared/Microsoft.Git.CredentialManager/GitConfigurationKeyComparer.cs

```diff
@@ -18,12 +18,16 @@ public class GitConfigurationKeyComparer : StringComparer
     {
         public static readonly GitConfigurationKeyComparer Instance = new GitConfigurationKeyComparer();
 
+        public static readonly StringComparer SectionComparer = OrdinalIgnoreCase;
+        public static readonly StringComparer ScopeComparer = Ordinal;
```

Comment:
Good catch, OAuth2 scopes are indeed case sensitive

Reply:
These are not OAuth2 scopes, but what I call "scopes" for Git configuration entries.. for example

`credential.https://github.com.useHttpPath` (`section.scope.property`)

(nit: I am aware that Git seems to call the different "levels" of config "scopes" - global, local etc)... naming is hard 😅